src/Diagnostics.Logger/ETWProvider/ETWMessageTemplates.cs

```diff
@@ -51,5 +58,8 @@ public static class ETWMessageTemplates
         public const string LogKustoTokenRefreshSummary = "Kusto Token Refresh Summary";
         public const string LogKustoQueryInformation = "Kusto Query Information";
         #endregion
+
+#pragma warning restore SA1600 // Elements should be documented
+#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
same here

Reply:
Use SuppressMessage for SA1600 and keep the `pragma` for CS1591